libs/blocks/video/video.css

```diff
@@ -5,4 +5,132 @@ a[href*='.mp4'].hide-video {
 video {
   max-width: 100%;
   height: auto;
+  object-fit: cover;
+}
+
+:is(.marquee, .aside.split) .pause-play-wrapper img.accessibility-control {
+  min-height: 40px;
+}
+
+.video-container {
+  display: flex;
+  position: relative;
+  height: 100%;
+  width: fit-content;
+  margin: auto;
+}
+
+:is(.aside, .marquee, .quiz-marquee) .video-container {
+  width: auto;
+}
+
+.brick-media .video-container {
+  width: 100%;
+  height: 100%;
+  border-radius: inherit;
+}
+
+.pause-play-wrapper {
+  display: flex;
+  width: fit-content;
+  margin: auto;
+}
+
+.video-container .pause-play-wrapper {
+  width: 45px !important;
+  height: 45px !important;
+  position: absolute;
+  bottom: 2%;
+  right: 2%;
+  margin: 0;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  z-index: 2;
+  cursor: pointer;
+}
+
+.video-container .pause-play-wrapper .offset-filler {  
+  display: inherit;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  border-radius:inherit;
```

Comment:
```suggestion
  border-radius: inherit;
```

Reply:
do we have a preferred css formatter(vscode extension) that i can make use of?

Reply:
Yes, we're using https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint